README.md

```diff
@@ -62,6 +62,23 @@ async function bootstrap() {
 bootstrap();
 ```
 
+By default, Nest's logger is disabled when using the `CommandFactory`. It's possible to provide it though, as the second argument to the `run` function. You can either provide a custom NestJS logger, or an array of log levels you want to keep - it might be useful to at least provide `['error']` here, if you only want to print out Nest's error logs.
+
+```ts
+import { CommandFactory } from 'nest-commander';
+import { AppModule } from './app.module';
+import { LogService } './log.service';
```

Comment:
Should we change this to Nest's logger?

```suggestion
import { Logger } from '@nestjs/common';
```
Or do you think `LogService` gets the point across well enough?

Reply:
Honestly, the only 2 use cases for me here are to either provide a custom logger, or an array of log levels from the example below.

This was copied from one other example in the docs, so it would also be a tiny bit more consistent

Reply:
Sounds good. Looks good to me! I'll get this merged and publish it this evening probably